### ISSUE TITLE
Feature Requests - V3.0

### ISSUE THREAD

**Issue by JeramyNS** [MyWorkAus/WP-Remote-WordPress-Plugin#167]
We have quite a few features planned for the next major release of WP Remote.

If you have any feature requests for the next version please feel free to comment below.

---
We will update this issue shortly to include a list of all features currently under review.

**Comment by Levdbas** [comment]
Check my feature request: #166 
Bad updates or deactivated plugins are the biggest issues I encounter while managing around 40 sites with WP-remote. This is not due WP-remote fault but faulty hosting in most cases. A check if the plugin is reactivated without failing would really help out!

**Comment by badcat** [comment]
**UI: Site List Positioning** // Does it make sense to allow manual list positioning (not just alphanumberic)  of sites -- and groups. Drag to reorder, in any order.
*** Export Site Info** // WP Core Version, Plugins, Themes; all with active status, version info

**Comment by CupOfTea696** [comment]
- Email notifications when WordPress version is out-of-date & optionally when plugins, themes, ... are out-of-date.
- Better error handling in the UI, I've seen some errors pop up in places where they shouldn't (see screenshot)
- Reordering sites: the grab icon's clickable area is rather small, I often click next to it and start selecting stuff instead of moving the site. Maybe include the favicon in the area that can be clicked to drag?
- Reordering groups in the same way you can re-order sites.
- Sorting sites & sites within groups

Screenshot 1:
![Screenshot 1](http://imgh.us/Screen_Shot_2017-05-25_at_16.45.43.png)

**Comment by wpstudio** [comment]
- Search enhancements that would allow me to search for a name of a plugin or theme and WPRemote would return all the sites that have that plugin/theme installed.
- Ability to move/drag groups of sites to a different order (up/down)... in addition to the current ability to reorder single sites.
- Integrate the "Notes" into the Site History list so that Notes can be exported along with the CSV export of activity.

**Comment by mrjamiebowman** [comment]
I would love to see an API.

This would result in mobile apps and I could integrate WP Remote into my custom client management portal.

**Comment by CupOfTea696** [comment]
Wordfence integration would be cool too.

**Comment by HugoGon** [comment]
Hi  @JeramyNS !

• Having the same issue as reported by @Levdbas, about deactivated plugins after updated through wpremote, or in rare cases currupted updates. (at the moment i just use wpremote as a warning system for outdated plugins/core installs, i don't have the trust to update from there, due to those cases in the past)
• Also would be nice to be able to change the /admin url in each website (in most of my cases i change it to something else).

Keep up the great work.
Cheers
Hugo

**Comment by n2rga** [comment]
Make the plugin php7 compatible.
these are the errors I get from PHP Compatibility Checker

```
FILE: /home/XXXXXXXXXX/public_html/wp-content/plugins/wpremote/wprp.hm.backup.php
---------------------------------------------------------------------------------------------------------------
FOUND 18 ERRORS AFFECTING 18 LINES
---------------------------------------------------------------------------------------------------------------
 1000 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1003 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1008 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1011 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1014 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1023 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1025 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1956 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1960 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1961 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1965 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1974 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1977 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1978 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1991 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 1992 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 2009 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
 2050 | ERROR | Extension 'mysql_' is deprecated since PHP 5.5 and removed since PHP 7.0 - use mysqli instead.
---------------------------------------------------------------------------------------------------------------
```

**Comment by JeramyNS** [comment]
Hey @n2rga it is already php7 compatible for the most part.
There is a legacy backup function that will be flagged in your php7 compatibility checker but it's essentially a false positive. I will however be fixing this up shortly to resolve these false positives.

@mrjamiebowman there is an api already available, check out the link below for more information:
https://wpremote.com/api-docs/

**Comment by Levdbas** [comment]
Hi @JeramyNS , glad to hear I am not the only once facing the update issues. In my case it only happens with more budget orientated hosting and in combination with large plugins. Do you experience the same?

**Comment by n2rga** [comment]
@JeramyNS
Thank You
Here's a feature request: I use a plug in to change the login URL and it blocks access to the /wp-admin/ URL would be great if we are able to change the URL for admin to our login URL for the site since the admin section is blocked by the plugin and url for login changed.

**Comment by JulieKuehl** [comment]
How about the ability to view the update details? Maybe by hovering/clicking on the orange version info area?

**Comment by badcat** [comment]
@JulieKuehl Yep - good one.  I'd forgotten I'd asked for this years ago. Would be nice to see the changelog and a link to the page on WordPress.org as available. Some plugins/themes do not live on WP's site, so what would happen then?

**Comment by badcat** [comment]
Would be nice to have the Site Listings (Left Side) as well as the Site Details (Right Side) as two _Separate Scrollable_ areas.  This way we dont have to needlessly scroll all the way up or down if there are a lot of sites shown in the left panel.
